examples/tree_2d_dgsem/elixir_navier_stokes_lid_driven_cavity.jl

```diff
@@ -0,0 +1,77 @@
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the ideal compressible Navier-Stokes equations
+
+equations = CompressibleEulerEquations2D(1.4)
+# Note: If you change the Navier-Stokes parameters here, also change them in the initial condition
+# I really do not like this structure but it should work for now
+equations_parabolic = CompressibleNavierStokesEquations2D(equations, Reynolds=1000, Prandtl=0.72,
+                                                          Mach_freestream=0.1)
+
+# Create DG solver with polynomial degree = 3 and (local) Lax-Friedrichs/Rusanov flux as surface flux
```

Comment:
Out of curiosity, did anyone ever try `flux_hll` for the Navier-Stokes tests?

Reply:
Resolving; issue created https://github.com/trixi-framework/Trixi.jl/issues/1196